orca_detector/orca_params.py

```diff
@@ -0,0 +1,28 @@
+"""
+Global parameters for the OrcaVGGish model.
+
+W251 (Summer 2019) - Spyros Garyfallos, Ram Iyer, Mike Winton
+"""
+
+# Classification params
+CLASSES = ['KillerWhale', 'FrasersDolphin']
+
+# Number of classes needs to account for "Other"
+NUM_CLASSES = len(CLASSES) + 1
+
+# Weighting to account for imbalance when calculating loss
+#TODO: update weights based on observed balance
+CLASS_WEIGHTS = {0: 1., 1: 1., 2: 1.}
+OPTIMIZER = 'sgd'
+LOSS = 'categorical_crossentropy'
+
+# Model training
+EPOCHS = 5
+BATCH_SIZE = 32
+
+# IMPORTANT: it's not straightforward to reduce this below 5, due to dependencies
+# between params.NUM_FRAMES and params.EXAMPLE_WINDOW_SECONDS (I think).  It will
+# take some debugging to work with shorter samples, but it may be worthwhile since
+# our dataset has quite a few samples < 5 sec.
```

Comment:
Let's verify this. 
Also, our samples contain the "raw" files too, which are very big unprocessed "sparse" files (mostly noise). We definitely need to calibrate our max limit

Reply:
I tried dropping to 2 seconds, but then got dimensionality mismatch errors when feeding to the model.  I'll file an issue to track this, as I think it's important to figure out.  But it looked like it would take enough work that I didn't want to try to tackle it in this PR.

Re: your second point, we should discuss how we want to better label the samples.  I think we may need to manually label them to find the "real" samples within the noise.  Maybe the audio annotation tool that was included in HW8?  I'll file an issue for that, too.

Reply:
That's a scary idea, my hope was that the small samples will carry the most information. Let's iterate once to see what's our accuracy, we might end up being lucky